format/Schema.fbs

```diff
@@ -134,11 +134,20 @@ table FixedSizeBinary {
 table Bool {
 }
 
+/// Exact decimal value represented as an integer value in two's
+/// complement. Currently only 128-bit (16-byte) integers are used but this may
```

Comment:
endianness?

Reply:
This should use the endianness indicated by the schema, I believe (like the other data types)?